src/plugin.cc

```diff
@@ -75,6 +77,11 @@ class OurGlueCompiler : public spicy::zeek::GlueCompiler {
 };
 
 plugin::Zeek_Spicy::Plugin::Plugin() {
+    // This ensures version compatibility at dlopen() time  by requiring the
+    // versioned symbol to be present.
+    _spicy_version = HILTI_VERSION_FUNCTION();
```

Comment:
This seems to be the only aspect requiring >=spicy-1.6.0 and I see you already pulled that into its own commit. Just wondering, do we want to backport the other two changes into the spicy-plugin version we pull into the Zeek LTS release?

Reply:
Actually wondering if we shouldn't make this conditional on the Spicy version we are running against (e.g., on the preprocessor level). This should simplify what can go into Zeek. As is we cannot publish any version of the plugin until spicy-1.6.0 is released.

Reply:
I'd say let's not backport into the Zeek release branch. But I can put just an `#ifdef` around this, then we can decide what/when to pull upstream.

Reply:
Sounds good, that should also allow us running CI against more Spicy versions.